clouddriver-aws/src/test/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/ResizeAsgAtomicOperationUnitSpec.groovy

```diff
@@ -110,8 +109,38 @@ class ResizeAsgAtomicOperationUnitSpec extends Specification {
     capacity(0, 2, 3)    | 1          | 2          | 3              || true
   }
 
+  void "should support resize requests with only some fields to update"() {
+    def mockAutoScaling = Mock(AmazonAutoScaling)
```

Comment:
This should be in a `given:` block (or `setup:` as this specification appears to be using)

Looks like this setup is duplicated a bit, could extract to class-level:

```
class ResizeAsgAtomicOperationUnitSpec extends Specification {

  def amazonAutoScaling = Mock(AmazonAutoScaling)
  def amazonClientProvider = Mock(AmazonClientProvider) {
    _ * getAutoScaling(_, _, true) >> { return amazonAutoScaling }
  }

  def setupSpec() {
    TaskRepository.threadLocalTask.set(Mock(Task))
  }

  ...
```

Reply:
good point, done in 6fbc3d7d7